pyaedt/edb_core/stackup.py

```diff
@@ -1714,9 +1743,12 @@ def _export_xml(self, file_path):
         return True
 
     @pyaedt_function_handler
-    def import_stackup(self, file_path):
+    def load(self, file_path):
         """Import stackup from a file. The file format can be XML, CSV, or JSON.
 
+        .. deprecated:: 0.6.61
+           Use :func:`import` instead.
```

Comment:
@dcrawforAtAnsys I thought we were supposed to use `load()` instead of `import_stackup()`.
So it might not be deprecated. Just to be sure.
```suggestion
```